### PR TITLE
Significantly Buffs Advanced Healing Chemicals

### PR DESCRIPTION
**Pull request by Isy232**
## About The Pull Request

Significantly buffs salicyclic acid, oxandrolone and atropine.

## Why It's Good For The Game

Salicyclic acid, and its burn damage counterpart, oxandrolone, are utter shit for how hard they are to make.

Someone walks into medbay with 80 brute damage, a fairly regular case. A doctor has 2 treatment options, salicyclic acid and bicardine.

Lets do some math:
- Bicardine heals 2 brute damage per tick. This means the bicardine will take only 40 ticks to fully heal this persons brute damage.
- Salicyclic acid heals 4 brute damage per tick while they have above 80 brute damage, so 8 ticks will be spent healing 4 brute damage per tick leaving 48 brute damage. Salicyclic acid then only heals 0.5 brute damage per tick, spending a whopping 96 FUCKING ticks to heal the remaining damage.

So the so called "superior" healing medicine that is complicated to mix takes 2.5 times longer to heal the damage.

THE ADVANCED BRUTE HEALING CHEMICAL TAKES 2.5 FUCKING TIMES LONGER TO HEAL 80 DAMAGE THAN THE SIMPLEST ONE.

This is the same for oxandrolone.

Now, atropine.

- Atropine before buff only healed 2 of each damage type per tick with the exception of 5 oxygen damage. It also only did this while you were in crit.

- Atropine is the second hardest to mix healing chemical in the game, yet its next to useless because dropping them into cryo or a sleeper is far more effective because dropping those don't loose all utility once they surpass crit.

My changes:
- Above 20 damage of brute or burn, they will heal 5 per tick. Under 20 damage they wil only heal 1 per tick.

- Atropine now heals 4 of each damage type (still 5 oxyloss though) and only stops being effective at 40hp.

Of note: Atropine also causes dizzyness and jittering while it is in your system. This makes it a very bad idea to use in active combat, hence why I think buffing it so hard is justified.

I also think this will be a good change as it will encourage chem man to make healing chemicals instead of bombs because the healing chemicals he can make are actually relevant and not worse than what can commonly be found in medbay

## Changelog
:cl:
balance: Significantly buffs the "superior" healing chemicals salicyclic acid, oxandrolone and atropine.
/:cl:

### CODE REVIEW

**Comment by Sinmero** [comment]
I think that all chems should have their own use and not just replace each other. (yeah synthflesh im looking at you)

> Salicyclic acid and oxandrolone now heal 5 damage per tick while below 20 damage and only 1 damager per tick while above 20.

For sali acid /oxandrolone i suggest lowering the threshold to 25 damage but leaving 4 healing per tick and 0.5 when above 25 damage (like tg did).

> Atropine now heals 4 of each damage type (still 5 oxyloss though) and only stops being effective at 40hp.

As for atropine i think it is a HUGE overkill. Currently atropine heals for 11 total damage / tick! Its meant for pulling patients out of crit / stabilizing defibbed patients. (I got defibbed people on their feet really fast with atropine/sali acid/oxandro mix)

Now it would heal for <b>17 total damage/tick up to 40 hp!</b>(way better healing than earthblood with its brain damage) Which is insane by itself. With metabolism rate 0.1 (kelotane has 0.4 as a comparison) you can just inject yourself and go tank tons of damage like its nothing which makes it a really good combat chem even with screen shaking. 
Also atropine pens are available in advanced medkits and at box medbay roundstart. 
Medbay is easy as it is dont want to make it even easier.

I suggest leaving atropine healing as it is and making the recipe more simple.

**Comment by Isy232** [comment]
> For sali acid /oxandrolone i suggest lowering the threshold to 25 damage but leaving 4 healing per tick and 0.5 when above 25 damage (like tg did).

Let me do some more math, again using the situation of someone coming into medbay with 80 brute damage.

- Bicardine takes 40 ticks to fully heal that 80 brute damage.

- Buffed salicyclic acid takes 12 ticks to heal 60 damage, then 20 more ticks to heal off the remaining 20 damage. So in totaly Salicyclic acid takes 32 ticks to heal 80 damage.

I still think this is balanced.

Comparing to /tg/ is difficult because /tg/ nerfed all of their basic chems to make healing more difficult. Sali has to compete with bicardine, synthflesh and styptic powder here, hence I think copying what /tg/ did will make sali remain near useless.

Now, for atropine. I also stand by this and let me give my reason why:

- Lets say John McSecson dies to a cult attack, you apply 1 synthflesh patch to get him revivable and then you apply a dose of atropine.

- John had 160 brute damage, 20 burn damage and (when revived) will have about 100 oxygen loss damage.

Now more math:
- In this case Atropine would take 30 ticks to get John out of crit and up to 40hp, leaving 40 brute damage to be dealt with by other means.

- Also, Atropine is also almost as hard to mix as pentic acid. 

So, overall I think buffing atropine this hard is fair. Other chems are much more suited to use in combat and atropine is rather complex to make. Advanced first aid kits cost 600 credits so only really heads of staff can buy them and those only have 2 atropine autoinjectors with 10u each.

**Comment by Sinmero** [comment]
I get your point with math but i dont think that advanced chems meant to be superior to regular chems in every way.
Why would you want to solve all the problems just by smacking some atropine and a brute patch on a guy (making atropine is not that hard once you put the recipes in the chem). 
Feels like those major buffs will make medbay even more casual (like splashing yourself with synthflesh in not casual enough).

We have loads of chemicals, sleepers, stasis beds, surgeries and cryo.
We should try to make medbay gameplay more complex and interesting. Chems should be situational and used in combinations.
Medbay will stop being fun when someone setups buffed atropine factory or synthflesh factory as it is now.

As i said before atropine is meant to pull people out of crit and not to be the ultimate healing chem. Its a good idea to make it easier to produce so people will use it more often.


Synthflesh bad

**Comment by Isy232** [comment]
> I get your point with math but i dont think that advanced chems meant to be superior to regular chems in every way.
> Why would you want to solve all the problems just by smacking some atropine and a brute patch on a guy (making atropine is not that hard once you put the recipes in the chem).
> Feels like those major buffs will make medbay even more casual (like splashing yourself with synthflesh in not casual enough).
> 
> We have loads of chemicals, sleepers, stasis beds, surgeries and cryo.
> We should try to make medbay gameplay more complex and interesting. Chems should be situational and used in combinations.
> Medbay will stop being fun when someone setups buffed atropine factory or synthflesh factory as it is now.
> 
> As i said before atropine is meant to pull people out of crit and not to be the ultimate healing chem. Its a good idea to make it easier to produce so people will use it more often.
> 
> Synthflesh bad

I completely understand your point, but what I am trying to do is escalate these chems to the level that others are at. If these chems are not on the same level as synthflesh they will continue to be ingorned.

**Comment by Chessus** [comment]
> Salicyclic acid and oxandrolone now heal 5 damage per tick while below 20 damage and only 1 damager per tick while above 20.

Just the wording of this, I think it's more clear to say that above 20 damage of brute or burn, they will heal 5 per tick. Under 20 damage they wil only heal 1 per tick.

**Comment by Isy232** [comment]
> > Salicyclic acid and oxandrolone now heal 5 damage per tick while below 20 damage and only 1 damager per tick while above 20.
> 
> Just the wording of this, I think it's more clear to say that above 20 damage of brute or burn, they will heal 5 per tick. Under 20 damage they wil only heal 1 per tick.

Yeah, you're right, my bad.

**Review by PowerfulBacon — DISMISSED** [review]

**Comment by PowerfulBacon** [comment]
If you combine the healing chems they are going to be super op

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -799,10 +799,10 @@
...
 /datum/reagent/medicine/atropine/on_mob_life(mob/living/carbon/M)
-	if(M.health <= M.crit_threshold)
-		M.adjustToxLoss(-2*REM, 0)
-		M.adjustBruteLoss(-2*REM, 0)
-		M.adjustFireLoss(-2*REM, 0)
+	if(M.health <= 40)
+		M.adjustToxLoss(-4*REM, 0)
+		M.adjustBruteLoss(-4*REM, 0)
+		M.adjustFireLoss(-4*REM, 0)
 		M.adjustOxyLoss(-5*REM, 0)
```
Comment (PowerfulBacon) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: This shouldn't be below 40 health, that defeats the whole point of a chemical for someone in critical condition

**Comment by Chessus** [comment]
> If you combine the healing chems they are going to be super op

True, but I think they won't be used frequently if synthflesh  remains unchanged with it's instant heal. It really needs to be an admin-forced change, not vote as players never vote on sf nerfs.

**Comment by Rukofamicom** [comment]
This is awful. The last thing medbay needs is even more powerful chems when they're already so powerful overall. 

> True, but I think they won't be used frequently if synthflesh remains unchanged

And synthflesh should not remain unchanged. Medbay is so marginalized as it is.

**Comment by Isy232** [comment]
> If you combine the healing chems they are going to be super op

I'll make it to 80 so it justs gets you out of crit

**Comment by Isy232** [comment]
Okay, so I nerfed Atropine so it bring you to just out of crit at -80.

**Comment by Isy232** [comment]
I no longer want this much of a buff and the changes are being moved to #2995.

**Pull request closed without merging**